### PR TITLE
[UNR-1698][MS] Updating docs regarding receptionistHost command line …

### PR DESCRIPTION
**Pull request by MatthewSandfordImprobable**
…arg.

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Adding documentation for [UNR-1698]

[UNR-1698]: https://improbableio.atlassian.net/browse/UNR-1698

### CODE REVIEW

File: SpatialGDK/Documentation/content/map-travel.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ PlayerController->ClientTravel(TravelURL.ToString(), TRAVEL_Absolute, false /*bS
 
 ## Default connection flows
 #### Play In Editor Clients
-Launching a [PIE](https://docs.unrealengine.com/en-us/GettingStarted/HowTo/PIE#playineditor) client from the editor will automatically attempt to connect to a local SpatialOS deployment on your machine by using the receptionist flow. It uses the URL `127.0.0.1` This is for quick editing and debugging purposes.
+Launching a [PIE](https://docs.unrealengine.com/en-us/GettingStarted/HowTo/PIE#playineditor) client from the editor will automatically attempt to connect to a local SpatialOS deployment on your machine by using the receptionist flow. It uses the URL `127.0.0.1` This is for quick editing and debugging purposes. The start up PIE client URI can be changed using the command line argument "receptionistHost <URL>". This allows the developer to connect to a non local server.
```
Comment (improbable-valentyn) [inline SpatialGDK/Documentation/content/map-travel.md]:
```suggestion
Launching a [PIE](https://docs.unrealengine.com/en-us/GettingStarted/HowTo/PIE#playineditor) client from the editor will automatically attempt to connect to a local SpatialOS deployment on your machine by using the receptionist flow. It uses the URL `127.0.0.1` This is for quick editing and debugging purposes. The start up PIE client URI can be changed using the command line argument `receptionistHost <URL>`. This allows the developer to connect to a non local server.
```

Comment (improbable-valentyn) [reply SpatialGDK/Documentation/content/map-travel.md]: >It uses the URL `127.0.0.1` This is for quick editing and debugging purposes.
Looks strange, we should fix this wording as well (this was before your change)

**Comment by ernopp** [comment]
@improbable-valentyn  what do you think of this updated wording?

One thing I'm not clear on and I think we should explain in the docs is which command you apply the `receptionistHost <URL>` command line argument to

File: SpatialGDK/Documentation/content/map-travel.md [hunk 2]
```diff
@@ -48,7 +48,9 @@ PlayerController->ClientTravel(TravelURL.ToString(), TRAVEL_Absolute, false /*bS
 
 ## Default connection flows
 #### Play In Editor Clients
-Launching a [PIE](https://docs.unrealengine.com/en-us/GettingStarted/HowTo/PIE#playineditor) client from the editor will automatically attempt to connect to a local SpatialOS deployment on your machine by using the receptionist flow. It uses the URL `127.0.0.1` This is for quick editing and debugging purposes.
+Launching a [PIE](https://docs.unrealengine.com/en-us/GettingStarted/HowTo/PIE#playineditor) client from the editor will automatically attempt to connect to a local SpatialOS deployment on your machine using the receptionist flow. 
+
+By default the PIE client URL is set to `127.0.0.1` (your local machine). To enable connecting to a remote SpatialOS receptionist, the PIE client URL can be changed with the following command line argument: `receptionistHost <URL>`.
```
Comment (improbable-valentyn) [inline SpatialGDK/Documentation/content/map-travel.md]: @ernopp 
```suggestion
By default the PIE client URL is set to `127.0.0.1` (your local machine). To enable connecting to a remote SpatialOS receptionist, the PIE client URL can be changed by passing the following command line argument when running the editor: `receptionistHost <URL>`. 
```

**Comment by MatthewSandfordImprobable** [comment]
Is this good to go?

**Pull request merged**